stan/math/opencl/copy.hpp

```diff
@@ -160,7 +162,7 @@ inline matrix_cl<T> packed_copy(const std::vector<T>& src, int rows) {
     matrix_cl<T> packed(packed_size, 1);
     cl::Event packed_event;
     const cl::CommandQueue queue = opencl_context.queue();
-    queue.enqueueWriteBuffer(packed.buffer(), CL_FALSE, 0,
+    queue.enqueueWriteBuffer(packed.buffer(), opencl_context.in_order(), 0,
```

Comment:
I don't expect someone constructing a std::vector temporary and passing it directely into a function. But maybe we should also change this one.